go/pkg/cas/upload.go

```diff
@@ -68,8 +70,27 @@ func (c *Client) Upload(ctx context.Context, inputC <-chan *UploadInput) (stats
 		fsSem:   semaphore.NewWeighted(int64(c.FSConcurrency)),
 	}
 
+	// Initialize checkBundler, which checks if a blob is present on the server.
+	u.checkBundler = bundler.NewBundler(&uploadItem{}, func(items interface{}) {
+		// Handle errors and context cancelation via errgroup.
+		eg.Go(func() error {
+			return u.check(ctx, items.([]*uploadItem))
+		})
+	})
+	u.checkBundler.BundleCountThreshold = u.FindMissingBlobsBatchSize
```

Comment:
I've added comments why this can't happen. Basically digests are fairly small. We can hit this limit with over 100K digests, but that's too much for other reasons.